src/components/article-body/document-body.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ComponentFactory } from './component-factory';
+import { ArticleNode } from '../../interfaces/article-body-node';
+import { GlobalStyledDocumentBody } from './styles';
+
+export const DocumentBody = ({ content }: any) => {
+    return (
+        <GlobalStyledDocumentBody>
+            {[content].map((child: ArticleNode, index: number) => (
+                <ComponentFactory key={index} nodeData={child} />
```

Comment:
Don't use the index as the key as it could cause issues in React https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067